tenets/codelingo/psr1/camel-case-method-name/codelingo.yaml

```diff
@@ -0,0 +1,17 @@
+tenets:
+  - name: camel-case-method-name
+    flows:
+      codelingo/docs:
+        title: Camel Case Method Name
+        body: Method names MUST be declared in camelCase
+      codelingo/review:
+        comment: Method names MUST be declared in camelCase
+    query: |
+      import codelingo/ast/php
+
+      php.stmt_class(depth = any):
+        php.stmt_classmethod:
+          @review comment
+          php.identifier:
+            name as methodName
+            regex(/^[A-Z]/, methodName)
```

Comment:
Shouldn't you be looking for non alphabetic separators instead? Like underscores and dashes etc.